client/src/Components/PreNavBar.js

```diff
@@ -21,6 +29,33 @@ function PreNavBar() {
             component={Link}
             to="/signup">Sign Up</Button>
         </Toolbar>
+    
+    </span>
+  )
+
+  // If the user is logged in, show profile page and logout links
+  if (props.user) {
+    links = (
+      <span>
+        <li>
+        <Link to="/profile">Profile</Link>
+        </li>
+        <li>
+          <Logout onClick={handleLogout}/>
+        </li>
+      </span>
+    )
+  }
+
```

Comment:
Yeah, this is not how I would do this, we already have boolean logic for what shows up higher in the tree.